lib/rubocop/cop/mixin/factory_bot_mixin.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Helpers for detecting FactoryBot usage.
+    module FactoryBotMixin
+      extend NodePattern::Macros
+
+      def_node_matcher :factory_bot, <<~PATTERN
+        (send _ {:build :build_list :create :create_list} $sym)
```

Comment:
should we also match on `attributes_for` and `build_stubbed`?

https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#build-strategies